app/views/groups/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<nav class="category_nav">
+  <img src="https://img.icons8.com/ios-filled/50/null/menu--v1.png" alt="menu bar" />
+  <h1>CATEGORY</h1>
+  <img src="https://img.icons8.com/ios/50/null/search--v1.png"/>
+</nav>
+
+<div id="groups">
+  <% @groups.each do |group| %>
+    <%= link_to group_entities_path(group) do %>
+      <%= render group %>
+    <% end %>
+  <% end %>
+</div>
+<div class="btn_container">
+  <%= button_to "New Category", new_group_path, method: :get %>
+</div>
```

Comment:
- [ ] kindly make the `new Category` button to stay at the bottom of  the page as it is in the design also do same for show page
your design :point_down: 
![Screenshot from 2023-04-28 16-49-21](https://user-images.githubusercontent.com/66097371/235197134-5564f998-bb83-4008-950b-2d4ecbf7a4e4.png)
given design :point_down: 
![Screenshot from 2023-04-28 11-55-45](https://user-images.githubusercontent.com/66097371/235197213-3484e86e-71cc-4334-96f9-b8120f78fc73.png)